test_sub_download.py

```diff
@@ -0,0 +1,35 @@
+'''Unit tests of subtitle_downloader'''
+import unittest
+import subtitle_downloader as subdl
+import random
+import os
+
+class TestSubDownloader(unittest.TestCase):
+    '''Unit tests for subtitle_downloader methods'''
+
+    def test_get_subtitles(self):
+        '''Positive test for method get_subtitles'''
+        try:
+            subs = subdl.get_subtitles("76b4703d5ace081a4f5116157751e891")
+        except Exception as exep:
+            self.fail("Raised {} unexpectedly!".format(exep))
+        self.assertIn("Let's go home.", subs)
+
+    def test_get_hash(self):
+        '''Positive test for method get_hash'''
+        tmp_file_name = "tmp_file"
```

Comment:
You could use the tempfile module to create temporary files, instead of hardcoding the filename. See https://docs.python.org/3/library/tempfile.html#examples